captslog/ui/centralwidget.py

```diff
@@ -17,16 +17,25 @@ def _translate(context, text, disambig):
     def _translate(context, text, disambig):
         return QtGui.QApplication.translate(context, text, disambig)
 
-#Collection of all widgets
 class CentralWidget(QtGui.QWidget):
+	"""Set up the central GUI
```

Comment:
You don't setup CentralGUIWidget here (You do that at the constructor.
Just describe what class this is or name it.